src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 3px 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin: 0 auto 4px;
+    color: $color-dark;
+  }
+
+  &__product-code {
+    line-height: 14px;
+    font-size: 10px;
+    color: $color-lighter;
+    margin: 0 0 16px;
+  }
+
+  &__review {
+    display: flex;
+    height: 16px;
+    margin-bottom: 24px;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__review-number {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-dark;
+    margin-top: 16px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 0 16px;
+  }
+
+  &__price-name {
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-lighter;
+    margin: 0;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    color: $color-dark;
+    font-size: 16px;
+    line-height: 18px;
+    margin: 0;
+  }
```

Comment:
Fix everywhere according this rule from checklist
![image](https://github.com/mate-academy/layout_catalog/assets/70009205/a0f38ad0-14b2-4ced-9bcb-c8bbb9937e5e)

Reply:
not fixed 
![image](https://github.com/mate-academy/layout_catalog/assets/70009205/d1acb423-35ef-455c-8468-4644fd256b72)